.circleci/config.yml

```diff
@@ -62,31 +61,18 @@ jobs:
           command: npm run lint
       - run:
           name: Run Tests
-          command: npm t
-
-  lint-and-test-next-lts:
-    <<: *lint-and-test
-    executor: node-container-next-lts
-
-  test-built-app: &test-built-app
-    executor: node-container
-    steps:
-      - checkout
+          command: npm test
       - run:
-          name: Run Tests
+          name: Build app and run its tests
           environment:
-            MODE: 'ci'
+            MODE: 'local'
```

Comment:
This needs to be 'ci', we don't want to test a new template here (since we are not creating a new template anyway). Tests for the new template are run in the other job (dependency-check), whose environment is set in Circle CI